.github/workflows/main.yml

```diff
@@ -81,26 +81,33 @@ jobs:
           CC: ${{ matrix.config.compiler }}
           CFLAGS: -Wall
         run: |
-          autoreconf -fiv
-          ./configure --enable-werror
+          cmake -E make_directory cmake-build
+          cmake -S . -B cmake-build
```

Comment:
We still need a way to compile with `-Werror` so that the build breaks if we introduce new warnings.

Reply:
Also, it would be nice to build both Shareware and Registered executable and upload both artifacts.